<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GBSBizLiveBridger</class>
 <widget class="QWidget" name="GBSBizLiveBridger">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1238</width>
    <height>920</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lblPreview">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>637</width>
     <height>468</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">      background: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, 
           stop:0 #000000, stop:0.525 #434343, stop:1 #000000); 
     border-radius: 20px;
 </string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QFrame" name="frmConsole">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>10</y>
     <width>456</width>
     <height>380</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#F9F9F9;
border: 1px solid  #FFFFFF;
border-radius: 16px;
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QLabel" name="lblLiveSource_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#F9F9F9;
border: 1px solid  #FFFFFF;
border-radius: 16px;
font-size: 24px;
</string>
    </property>
    <property name="text">
     <string>控制台</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>150</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>150</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>150</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>160</y>
      <width>150</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>260</y>
      <width>150</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>260</y>
      <width>150</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frmLiveSource">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>500</y>
     <width>281</width>
     <height>412</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#F9F9F9;
border: 1px solid  #FFFFFF;
border-radius: 16px;
border-top-left-radius: 10px;
border-top-right-radius: 10px;
border-bottom-left-radius: 0px;
border-bottom-right-radius: 0px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QLabel" name="lblLiveSource">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>66</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#F9F9F9;
border: 1px solid  #FFFFFF;
border-radius: 16px;
</string>
    </property>
    <property name="text">
     <string>直播间源</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frmLocalSource">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>500</y>
     <width>585</width>
     <height>412</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#F9F9F9;
border: 1px solid  #FFFFFF;
border-radius: 16px;
border-top-left-radius: 10px;
border-top-right-radius: 10px;
border-bottom-left-radius: 0px;
border-bottom-right-radius: 0px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>0</y>
      <width>591</width>
      <height>411</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horiLocalSourceLayout"/>
   </widget>
  </widget>
  <widget class="QFrame" name="frmAudioPannel">
   <property name="geometry">
    <rect>
     <x>966</x>
     <y>500</y>
     <width>211</width>
     <height>412</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#F9F9F9;
border: 1px solid  #FFFFFF;
border-radius: 16px;
border-top-left-radius: 10px;
border-top-right-radius: 10px;
border-bottom-left-radius: 0px;
border-bottom-right-radius: 0px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>211</width>
      <height>411</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="vertAudioLayout"/>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
